src/tools/x/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "x"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
this doesn't actually need to be updated, the *behavior* of x isn't changing, only the code